MIGRATING.md

```diff
@@ -2,6 +2,315 @@
+- `JsonWebKey setX5t(String x5t)`
+- `JsonWebKey setX5tS256(String x5tS256)`
+- `JsonWebKey setX5u(String x5u)`
+
+### Package `com.okta.sdk.resource.application.OpenIdConnectApplicationIdpInitiatedLogin`
+
+This is a newly created interface with methods listed below.
+
+- `List<String> getDefaultScope()`
+- `OpenIdConnectApplicationIdpInitiatedLogin setDefaultScope(List<String> defaultScope)`
+- `String getMode()`
+- `OpenIdConnectApplicationIdpInitiatedLogin setMode(String mode)`
+
+### Package `com.okta.sdk.resource.application.OpenIdConnectApplicationSettingsClient`
+
+Below methods have been added.
+
+- `OpenIdConnectApplicationIdpInitiatedLogin getIdpInitiatedLogin()`
+- `OpenIdConnectApplicationSettingsClient setIdpInitiatedLogin(OpenIdConnectApplicationIdpInitiatedLogin idpInitiatedLogin)`
+
+### Package `com.okta.sdk.resource.application.SamlApplicationSettingsSignOn`
+
+Below methods have been added.
+
+- `SingleLogout getSlo()`
+- `SpCertificate getSpCertificate()`
+- `SamlApplicationSettingsSignOn setSlo(SingleLogout slo)`
+- `SamlApplicationSettingsSignOn setSpCertificate(SpCertificate spCertificate)`
+
+### Package `com.okta.sdk.resource.application.SamlApplicationV1`
+
+The Interface has been removed.
+The Interface `com.okta.sdk.resource.application.SamlApplication` should be used instead.
+
+
+### Package `com.okta.sdk.resource.application.SingleLogout`
+
+This is a newly created interface with methods listed below.
+
+- `Boolean getEnabled()`
+- `String getIssuer()`
+- `String getLogoutUrl()`
+- `SingleLogout setEnabled(Boolean enabled)`
+- `SingleLogout setIssuer(String issuer)`
+- `SingleLogout setLogoutUrl(String logoutUrl)`
+
+### Package `com.okta.sdk.resource.application.SpCertificate`
+
+This is a newly created interface with methods listed below.
+
+- `List<String> getX5c()`
+- `SpCertificate setX5c(List<String> x5c)`
+
+### Package `com.okta.sdk.resource.authorization.server.AuthorizationServerPolicy`
+
+Below method has been renamed to keep naming consistency.
+- Renamed `deletePolicy` to `delete`
+
+Below method has undergone a signature change.
+- `void deletePolicyRule(String ruleId)` signature changed to `void deletePolicyRule(String authServerId, String ruleId)`
+
+Below method has undergone a signature change and renamed to keep naming consistency.
+- `AuthorizationServerPolicy updatePolicy(String authServerId, AuthorizationServerPolicy authServerPolicy)` changed to `AuthorizationServerPolicy update(String authServerId)`
+
+### Package `com.okta.sdk.resource.authorization.server.policy.AuthorizationServerPolicyRule`
+
+Below method has been renamed to keep naming consistency.
+- Renamed `deletePolicyRule` to `delete`
+
+### Package `com.okta.sdk.resource.inline.hook.InlineHookChannelConfig`
+
+Below methods have been added.
+
+- `String getMethod()`
+- `InlineHookChannelConfig setMethod(String method)`
+
+### Package `com.okta.sdk.resource.linked.object.LinkedObject`
+
+Below methods have been removed.
```

Comment:
Please add a note explaining why and what the callers already using this should do now?